examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running the Quantum Volume Algorithm\n",
+    "This example walks through the steps of running the quantum volume algorithm on square matrices. It is intended to mirror Algorithm 1 of https://arxiv.org/pdf/1811.12926.pdf. In general, we will generate a model circuit, classically compute its Heavy Output Group, then run various samplers (currently, ideal and noisy simulators) to evaluate how often they generate Heavy results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cirq\n",
+    "from examples.advanced import quantum_volume\n",
+    "import numpy as np\n",
+    "\n",
+    "# Some set-up variables. Feel free to play with these.\n",
+    "num_repetitions = 11 # This is supposed to be >= 100.\n",
+    "depths = range(2, 5) # The depths and number of qubits\n",
+    "\n",
+    "# Here is the important set-up: The samplers. These are what will be\n",
+    "# run on the generated circuit, and then evaluated.\n",
+    "class SamplerConfig:\n",
+    "    def __init__(\n",
+    "        self,\n",
+    "        *,  # Forces keyword args.\n",
+    "        sampler: cirq.Sampler,\n",
+    "        label: str,\n",
+    "        color: str,\n",
+    "        marker: str = None):\n",
+    "            self.sampler = sampler\n",
+    "            self.label = label\n",
+    "            self.color = color\n",
+    "            self.marker = marker      \n",
+    "            # Keep track of the probabilities.\n",
+    "            self.probabilities = [0] * len(depths)\n",
```

Comment:
The existence of this field directly contradicts the notion that this is a config.

Reply:
Changed this a bit.